src/renderer/components/wallet/txs/TxForm.style.tsx

```diff
@@ -72,3 +72,38 @@ export const ButtonLinkIcon = styled(SelectOutlined)`
 export const Text = styled('span')`
   color: ${palette('text', 2)};
 `
+
```

Comment:
could you pls add additional styles for this form to "stretch" it for the whole screen ? (all needed style changes are displayed at the screenshot)
![image](https://user-images.githubusercontent.com/67793194/105970890-eb123080-609a-11eb-9898-894eb695f3a5.png)

Reply:
We don't do it in other forms (see send forms), let's keep the same here.

Reply:
ok. but we need to fix them all in future to match figma-designs
![image](https://user-images.githubusercontent.com/67793194/105978012-1c8efa00-60a3-11eb-8aa0-ab53bd8ff7ac.png)